ccm_web/server/src/api/api.controller.ts

```diff
@@ -157,7 +158,7 @@ export class APIController {
     return result
   }
 
-  @UseInterceptors(InvalidTokenInterceptor)
+  @UseInterceptors(InvalidTokenInterceptor, new TooManyResultsInterceptor('courses'))
```

Comment:
what is the need to add the `new` keyword here? is there any difference?

Reply:
The `new` just means we're passing an instance instead of a class, which I need to do because I want to make the plural entities string vary from method to method. Sounds like from the documentation that either will work; see https://docs.nestjs.com/interceptors#binding-interceptors